casper/src/main/scala/coop/rchain/casper/CreateBlockStatus.scala

```diff
@@ -1,13 +1,19 @@
 package coop.rchain.casper
 
+import cats.Monad
+import cats.implicits._
 import coop.rchain.casper.protocol.BlockMessage
 
 sealed trait CreateBlockStatus {
   def map(f: BlockMessage => BlockMessage): CreateBlockStatus = this
+  def mapF[F[_]: Monad](f: BlockMessage => F[BlockMessage]): F[CreateBlockStatus] =
```

Comment:
Again, `Monad` is a stronger constraint than you actually need.

Reply:
Yep, sorry